### ISSUE TITLE
not reporting all module licenses

### ISSUE THREAD

**Issue by jojastahl** [xz64/license-webpack-plugin#24]
Hello,
I created a new angular project with ng new lic-test.
I copied the following list of dependencies into package.json:
```javascript
    "@angular/animations": "~5.2.3",
    "@angular/cdk": "~5.1.1",
    "@angular/common": "~5.2.3",
    "@angular/compiler": "~5.2.3",
    "@angular/core": "~5.2.3",
    "@angular/forms": "~5.2.3",
    "@angular/http": "~5.2.3",
    "@angular/material": "~5.1.1",
    "@angular/platform-browser": "~5.2.3",
    "@angular/platform-browser-dynamic": "~5.2.3",
    "@angular/router": "~5.2.3",
    "@auth0/angular-jwt": "^1.0.0-beta.9",
    "classlist.js": "~1.1.20150312",
    "core-js": "~2.5.3",
    "hammerjs": "~2.0.8",
    "intl": "~1.2.5",
    "jsencrypt": "^2.3.1",
    "rxjs": "~5.5.6",
    "underscore": "~1.8.3",
    "web-animations-js": "~2.3.1",
    "zone.js": "~0.8.14"
```
Then I did npm install, ng build --prod
The resulting 3rdpartylicenses.txt only contains the modules 

- `core-js@2.5.3`
- `webpack@3.10.0`
- `zone.js@0.8.20`
- `@angular/core@5.2.5`  (without license text)

Why for example `@auth0/angular-jwt` is missing? It's MIT, it has a LICENSE file...

I'm on Windows 10, node version 8.9.1, npm 5.5.1

Is there anything I'm doing wrong?
Thank you.

**Comment by xz64** [comment]
The plugin doesn't check package.json. Instead it looks at the webpack output and reports licenses for packages which were imported by your source code. Are you still seeing any issue after importing a package in your source code and using the package?

**Comment by jojastahl** [comment]
In my real project, I'm using the `@auth0/angular-jwt` package for sure, but it doesn't show in the 3rdpartylicenses.txt.
In order to test in the simplified test case above, I modified the generated app.component.ts to import and use a function out of it as follows:
```javascript
import { Component } from '@angular/core';
import { JwtHelperService } from '@auth0/angular-jwt';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent {
  title = 'app';
  
  constructor(private _jwtService: JwtHelperService) {
    _jwtService.decodeToken("fjdgfdshj");
  }
}
```
But still, the package isn't mentioned in 3rdpartylicenses.txt.
I checked the geneated main bundle and searched for decodeToken function and found it there. So code from the package ends up in the generated output.

**Comment by xz64** [comment]
Fixed in version 1.1.2

Thanks for the bug report!
